common/src/primitives/amount.rs

```diff
@@ -42,6 +41,12 @@ impl From<u128> for Amount {
     }
 }
 
+impl From<Amount> for u128 {
```

Comment:
Let's discuss this tomorrow. I wanna know why simple addition of amounts isn't feasible.

Reply:
Addition is not the problem, random generation of an Amount within a given rand is the problem.

Reply:
Found a workaround. Removed this `From`.